### PR TITLE
LIR Parcel Metadata for Individual Counties

### PR DESCRIPTION
**Pull request by miriamseely**
This PR includes a schema summary file and a main metadata file for each of the 29 counties. I filled in the information using the SGID index and the metadata available on AGOL, please let me know if there is more information that needs to be filled in or if there are corrections that need to be made! Thanks everyone!

### CODE REVIEW

**Comment by jacobdadams** [comment]
Rick and I were talking about this today, and I'm thinking that for the `_schema.md` files we should just leave out any fields that aren't different/added. This would leave most of them mostly blank except for Salt lake (and maybe one other county, Rick can confirm this), which would then have one or two fields added/updated. This would make it easier in the future to change the base schema because we wouldn't have to repeat it in each county schema file.

@steveoh, can you think of a reason to keep the blank field headings in the individual county schema files that I've missed? Otherwise, I'd say lets clean the contents of those files. I know that undoes a lot of work, and I apologize :)

**Comment by steveoh** [comment]
> can you think of a reason to keep the blank field headings in the individual county schema files that I've missed? 

A reason I would say to leave them is that it's the same convention you are using for merging data with the other files. It might be confusing to have multiple merging strategies. 

My suggestion would be to keep it as simple as possible and if you think that change is the right way to do it, then do it also with the metadata files.

**Comment by jacobdadams** [comment]
> A reason I would say to leave them is that it's the same convention you are using for merging data with the other files. It might be confusing to have multiple merging strategies.

I agree, it's different from the normal text-based metadata file that has a standard set of headings. I'm thinking that this is special enough to warrant being different because the schema files don't have a standard set of headings, and the headings (ie, field names) for parcels (and their "sub-schemas" amongst the counties) are going to be different than the headings for roads, etc.

**Comment by miriamseely** [comment]
> I'm thinking that this is special enough to warrant being different because the schema files don't have a standard set of headings, and the headings (ie, field names) for parcels (and their "sub-schemas" amongst the counties) are going to be different than the headings for roads, etc.

In that case, do you want me to go ahead and delete the headings in the schema files (except for Salt Lake, of course) and just leave the title and the ID for each of them? @jacobdadams

**Comment by jacobdadams** [comment]
Yeah, go ahead and nuke all those. Sorry for the extra work! Just trying to lay track before the train gets there :)

**Comment by rkelson** [comment]
I don't see any LIR parcel metadata that is unique with the exception of Salt Lake. The basic has a few more counties with unique attributes

**Comment by rkelson** [comment]
Sorry folks I've been running around taking calls and getting ready for my procedure

**Comment by steveoh** [comment]
Please make sure to document these conventions somewhere so our future selves will remember how we intend for this to work.

**Comment by miriamseely** [comment]
> I don't see any LIR parcel metadata that is unique with the exception of Salt Lake. The basic has a few more counties with unique attributes

For the Basic parcels (which I will be putting on a separate branch) do you want them to have the blank headings, or should I just make the files with the Title and ID (and then include the unique attributes where applicable)? @rkelson

**Review by rkelson — COMMENTED** [review]
Uintah LIR should have the same schema as all the other LIR parcels with the only exception being Salt Lake LIR that has the additional field _PROP_TYPE_

File: metadata/Cadastre/lir_parcels/duchesne_county_parcels__lir_schema.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Title
```
Comment (steveoh) [inline metadata/Cadastre/lir_parcels/duchesne_county_parcels__lir_schema.md:1]: this filename has an extra underscore

Comment (miriamseely) [reply metadata/Cadastre/lir_parcels/duchesne_county_parcels__lir_schema.md:1]: Good catch!

**Pull request merged**